index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="main-wrapper">
+      <div class="button-wrapper">
+        <h1>What is your Gender?</h1>
+        <input type="radio" name="inputs" id="input1" />
+        <label class="entry" for="input1"
+          ><div class="circle"></div>
+          <div class="entry-label">Male</div>
+        </label>
+        <input type="radio" name="inputs" id="input2" />
+        <label class="entry" for="input2"
+          ><div class="circle"></div>
+          <div class="entry-label">Female</div>
+        </label>
+        <input type="radio" name="inputs" id="input3" />
+        <label class="entry" for="input3"
+          ><div class="circle"></div>
```

Comment:
`>?`

Reply:
It is due to prettier problem probably @syedabdullahali

Reply:
it is prettier thing it drags the bracket to new line rest there is no issues in code @syedabdullahali